test/FunctionsV2/PlatformSpecificHelpers.FunctionsV2.cs

```diff
@@ -54,5 +55,37 @@ public static class PlatformSpecificHelpers
 
             return (JobHost)host.Services.GetService<IJobHost>();
         }
+
+        private static IWebJobsBuilder AddCorrectDurableTaskExtension(this IWebJobsBuilder builder, IOptions<DurableTaskOptions> options, string storageProvider)
```

Comment:
"Correct" feels a little odd. It implies that something might be incorrect. I would suggest keeping the previous name.